pkg/v1/google/list.go

```diff
@@ -98,12 +98,14 @@ func (l *lister) list(repo name.Repository) (*Tags, error) {
 	return &tags, nil
 }
 
+// Uploaded uses json.Number to work around GCR returning timeUploaded
+// as a string, while AR returns the same field as int64.
 type rawManifestInfo struct {
-	Size      string   `json:"imageSizeBytes"`
-	MediaType string   `json:"mediaType"`
-	Created   string   `json:"timeCreatedMs"`
-	Uploaded  string   `json:"timeUploadedMs"`
-	Tags      []string `json:"tag"`
+	Size      string      `json:"imageSizeBytes"`
+	MediaType string      `json:"mediaType"`
+	Created   string      `json:"timeCreatedMs"`
+	Uploaded  json.Number `json:"timeUploadedMs"`
```

Comment:
We intend for AR to return the same set of fields as GCR, so we can fix this issue on the server side rather than on the client.

Reply:
That's great! Until then, given that it's a minor change that handles both GCR and AR (and you could argue that json.Number is the appropriate type for a number returned in a JSON document...), we could leave this in place so gcrane works for early AR users?

Reply:
Seems reasonable to me!